Adafruit_AS7341.cpp

```diff
@@ -68,6 +68,8 @@ bool Adafruit_AS7341::begin(uint8_t i2c_address, TwoWire *wire,
  */
 bool Adafruit_AS7341::_init(int32_t sensor_id) {
 
+  (void)sensor_id;      // may be in the future
```

Comment:
Please change the comment here to `// to avoid compiler warning` to better flag that it's a hack to get around a compiler warn. This follows other libraries that have also done this.